megaphone-validator/index.js

```diff
@@ -0,0 +1,39 @@
+const compareAudioFiles = require('./lib/compare-audio-files');
+
+// Load environment variables
+require('dotenv').config();
+
+const programSlugString = process.env.SCPR_PROGRAM_SLUGS || '';
+
+exports.handler = (event, context, callback) => {
+    // Stops executing when the callback is fired.
+    context.callbackWaitsForEmptyEventLoop = false;
+
+    // Exit if there are no podcasts indicated in the env
+    if (!programSlugString) {
+        return false;
```

Comment:
If the point of entry is in this file, we should exit early if any of the required env variables like `MEGAPHONE_AUTH_TOKEN` or `MEGAPHONE_NETWORK_ID` are missing, since without those, this script wouldn't run to completion anyway, right?

Reply:
@nakedsushi Ah, yes you're right! I went ahead and added checks for those two as well so that the function can exit early if those aren't indicated